.adops/package-update.yml

```diff
@@ -87,15 +98,32 @@ steps:
 
   - script: |
       git fetch
-      git checkout $(buildSourceBranch)
-      git pull origin $(buildSourceBranch)
       git add .
       git commit -m "chore: bumped Mosaic packages [skip ci]"
-      git push origin $(buildSourceBranch)
+      
+      echo "Pushing to temporary branch: $(temporaryPushBranch) to evaluate successful status check"
+      git checkout -b $(temporaryPushBranch)
+      git push origin $(temporaryPushBranch)
+      
+      echo "Wait 5 minutes for the status check to finish"
+      sleep 300
```

Comment:
this feels a bit flakey - but also good enough for us to test the whole check-run process. eventually we could think of calling an API to get the current status of the check-run in a loop (with a timeout and fail if not completed within that)